interfaces/portal/src/app/pages/program-registration-debit-cards/program-registration-debit-cards.page.ts

```diff
@@ -82,6 +95,32 @@ export class ProgramRegistrationDebitCardsPageComponent {
 
   program = injectQuery(this.programApiService.getProgram(this.programId));
 
+  fspConfigurationProperties = injectQuery(
+    this.fspConfigurationApiService.getFspConfigurationProperties({
+      programId: this.programId,
+      configurationName: 'Intersolve-visa',
+    }),
+  );
+
+  readonly cardDistributionByMailEnabled = computed(() => {
+    const props: FspConfigurationProperty[] =
+      this.fspConfigurationProperties.data() ?? [];
+
+    const cardDistributionByMailProperty = props.find(
+      (property) =>
+        property.name ===
+        (IntersolveVisaFspConfigurationProperties.cardDistributionByMail as string),
+    );
+
+    return cardDistributionByMailProperty?.value === 'true';
```

Comment:
Wouldn't it be safer/cleaner/prettier to use a helper-method to check for "any positive value"? Like "`1`", "`TRUE`", "`Y`", etc?

For example using `stringbool()` from Zod?